README.md

```diff
@@ -313,7 +323,34 @@ You can then access the WebUIs of the TDP services through Knox:
 
 _Note: You can login to Knox using the `tdp_user` that is created in the next step._
 
-**Create Cluster Users**
+#### Livy
+
+Deploys Livy Server on the `[livy_server]` group hosts:
+
+```bash
+ansible-playbook deploy-livy.yml
+```
+
+The Livy Server can be accessed at https://edge-01.tdp:8998 After deployment, one can create a Spark session and interact with it through cURL:
+
+```bash
+# From edge-01.tdp
+sudo su tdp_user
+kinit -kt ~/tdp_user.keytab tdp_user@REALM.TDP
+
+# Create a session
+curl -k -u : --negotiate -X POST https://edge-01.tdp:8998/sessions \
+  -d '{"kind": "pyspark"}' -H 'Content-Type: application/json'
+# Get the session status (wait until it is "idle")
+curl -k -u : --negotiate -X GET https://edge-01.tdp:8998/sessions
+# Submit a snippet of code to the session
+curl -k -u : --negotiate -X POST https://edge-01.tdp:8998/sessions/1/statements \
+  -d '{"code": "1 + 1"}' -H 'Content-Type: application/json'
+# Get the statement result
+curl -sk -u : --negotiate -X GET https://edge-01.tdp:8998/sessions/1/statements/0
```

Comment:
I think it should be `/sessions/0` on first launch

Reply:
Yes probably